packages/experimental/Avatar/src/JSAvatar.helpers.ts

```diff
@@ -106,21 +110,22 @@ export function calculateEffectiveSizes(tokens: JSAvatarTokens): AvatarSizeConfi
   }
 }
 
-export function getRingThickness(thickness: RingThickness): number {
-  switch (thickness) {
-    case 'xSmall':
-      return 1;
-    case 'small':
-      return 1;
-    case 'medium':
-      return 2;
-    case 'large':
-      return 2;
-    case 'xlarge':
-      return 2;
-    case 'xxlarge':
-      return 4;
-    default:
-      return thickness < 0 ? 4 : thickness;
-  }
+export function getRingConfig(size: number): any {
+  if (size <= 48)
+    return {
```

Comment:
We should wrap these return statements in braces, as it's a multiline statement.

```
if (size <= 71) {
    return {
      size: size + sizeAdjustment.medium,
      stroke: globalTokens.stroke.width.thicker,
      innerStroke: globalTokens.stroke.width.thicker,
    };
}
```

Reply:
fixed